docs/lite.md

```diff
@@ -17,13 +17,13 @@ The Pluggable Modular Parsers provide the ability to customize and extend the da
 The Splunk platform is the destination for the syslog data. Splunk allows for comprehensive analysis, searching, and visualization of the processed data.
 
 ##  Data Flow
-Source systems send syslog data to SC4S Lite. The data may be transmitted via UDP, TCP, or RELP, depending on the system's capabilities and configurations.
+Source systems send syslog data to SC4S Lite. The data may be transmitted via UDP or TCP depending on the system's capabilities and configurations.
```

Comment:
Source systems send syslog data to SC4S Lite through UDP or TCP, depending on your system's capabilities and configurations.